File: python/019.py
# 19) Crie um algoritmo que leia o nome e as duas notas de um aluno, calcule a sua
# média e mostre na tela. No final, analise a média e mostre se o aluno teve ou
# não um bom aproveitamento (se ficou acima da média 7.0).

nome_aluno = input("Qual o nome do aluno: ")
nota_1 = float(input(f"Qual a primeira nota do aluno {nome_aluno}: "))
nota_2 = float(input(f"Qual a segunda nota do aluno {nome_aluno}: "))

media = (nota_1 + nota_2) / 2
if media > 7:
    print(f"O aluno {nome_aluno} ficou acima da média com {media} de média.")
else:
    print(f"O aluno {nome_aluno} ficou abaixo da média com {media} de média.")

File: javascript/019.js
/*
19) Crie um algoritmo que leia o nome e as duas notas de um aluno, calcule a sua
média e mostre na tela. No final, analise a média e mostre se o aluno teve ou
não um bom aproveitamento (se ficou acima da média 7.0).
*/

const media = (aluno, ...notas) => {
  let total = 0;
  for (let i = 0; i < notas.length; i++) {
    total += notas[i];
  }
  const media = total / notas.length;

  return `Média do aluno: ${aluno} é ${media}`;
};

console.log(media("Kaik", 10, 8.5));
